src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapGood.java

```diff
@@ -1,12 +1,14 @@
 package com.google.checkstyle.test.chapter3filestructure.rule332nolinewrap; //ok
 
+import static java.math.BigInteger.ZERO;
```

Comment:
your a lucky - https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/travis.sh#L22

as by its name "InputNoLineWrapGood.java"
this file should be correct (no violations) - so please put it to proper place, no changes in UTs expected, as no violations should be.
I it was my bad request to places it in wrong place.